data/json/item_groups.json

```diff
@@ -10240,5 +10240,93 @@
         [ "material_sand", 80 ],
         [ "chemistry_set", 5 ]
       ]
+    },
+    {
+      "type": "item_group",
+      "id": "roof_holdout",
+      "items": [
+        { "item": "grenade", "prob": 5 },
+        { "item": "binoculars", "prob": 5 },
+        { "item": "mre_maplesausage_box", "prob": 5 },
+        { "item": "mre_ravioli_box", "prob": 5 },
+        { "item": "mre_hashbrownbacon_box", "prob": 0 },
+        { "item": "mre_lemontuna_box", "prob": 5 },
+        { "item": "mre_asianbeef_box", "prob": 5 },
+        { "item": "ak47", "prob": 20 },
+        { "item": "akmmag", "prob": 20 },
+        { "item": "762_54R", "prob": 20 },
```

Comment:
AK-47 fires 7.62x39mm, not 7.62x54mm rounds.

Reply:
thanks, so much ammo to choose from and so little knowledge about guns...